src/pages/newPost/NewPost.jsx

```diff
@@ -20,6 +23,43 @@ const styles = {
 
 const NewPost = () => {
   const [images, setImages] = useState([]);
+  const [title, setTitle] = useState("");
+  const [text, setText] = useState("");
+  const [community, setCommunity] = useState('');
+
+  const {communities,loading} = useCommunities();
+
+  const navigate = useNavigate()
+  
+  const postCreateApiCall = useMutation(
+    `/communities/${community}/posts`,
+    {
+      method:"post"
+    }
+  );
+  
+  const CreatePostClick = ()=>{
+    postCreateApiCall({
+      data: {
+        title:title,
+        bodyText: text,
+      },
+    }).then(function(success){
+      alert("successfully create a post")
```

Comment:
alert?

Reply:
do you mean change alert to something else?

Reply:
I don't mind keeping it as alert.

Reply:
I used Chakra UI useToast() to replace the alerts in my PR